src/clr-angular/forms/select/options.ts

```diff
@@ -4,30 +4,30 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import {Component, ElementRef, HostBinding, Inject, Injector, OnDestroy, Optional} from "@angular/core";
+import {Component, ElementRef, HostBinding, HostListener, Inject, Injector, OnDestroy, Optional} from "@angular/core";
 import {Subscription} from "rxjs/Subscription";
 
 import {AbstractPopover} from "../../popover/common/abstract-popover";
 import {Point} from "../../popover/common/popover";
 import {POPOVER_HOST_ANCHOR} from "../../popover/common/popover-host-anchor.token";
 import {IfOpenService} from "../../utils/conditional/if-open.service";
+import {BlurEventDisableService} from "./providers/blur-event-disable.service";
 
 @Component({selector: "clr-options", templateUrl: "options.html", host: {"[class.clr-options]": "true"}})
 export class ClrOptions extends AbstractPopover implements OnDestroy {
-    // Have to declare this separately or TS complains that super call in the constructor
-    // should be before the if condition which we do not want in this case
-    protected ifOpenService: IfOpenService;
     private sub: Subscription;
+    private blurEventDisableService: BlurEventDisableService;
 
     constructor(injector: Injector, @Optional() @Inject(POPOVER_HOST_ANCHOR) parentHost: ElementRef,
-                ifOpenService: IfOpenService) {
+                ifOpenService: IfOpenService, blurEventDisableService: BlurEventDisableService) {
         if (!parentHost) {
-            throw new Error("clr-select-menu should only be used inside of a clr-select");
+            throw new Error("clr-options should only be used inside of a clr-select");
         }
         super(injector, parentHost);
 
         // Initialize Services
         this.ifOpenService = ifOpenService;
```

Comment:
This is still not needed, `AbstractPopover` already sets it. 😛 It also means you don't even need to inject the `IfOpenService` in the constructor here.